framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/deployment/KieServerOperatorDeployment.java

```diff
@@ -64,15 +70,15 @@ public void waitForScale() {
         }
     }
 
-    private Server getAssociatedServerObject(KieApp kieApp) {
-        return Arrays.asList(kieApp.getSpec().getObjects().getServers()).stream().filter(s -> s.getName().equals(getServiceName()))
+    private Server getAssociatedServerObject(Spec spec) {
+        return Arrays.asList(spec.getObjects().getServers()).stream().filter(s -> s.getName().equals(getServiceName()))
```

Comment:
Here I think we can use kieApp.getStatus().getApplied() as we are trying to get created objects

Reply:
Spec comes from kieApp.getStatus().getApplied() already. See line 48 and 61.